packages/webdriverio/tests/utils/interception/devtools.test.js

```diff
@@ -1,7 +1,19 @@
-import { canAccess } from '@wdio/utils'
-
 import NetworkInterception from '../../../src/utils/interception/devtools'
 
+const fse = jest.requireActual('fs-extra')
```

Comment:
Why do we import the actual package? I think it would be better to use a mock version throughout the whole test.

Reply:
I need this one to read the file